Dockerfile

```
@@ -1,8 +1,24 @@
 FROM python:3.13-slim
 WORKDIR /opt/app
 
+ENV APP_USER=dvtools
+ENV APP_UID=40088
+
+ENV APP_GROUP=dpgdil
+ENV APP_GID=105
+
+RUN groupadd --system --gid $APP_UID $APP_USER \
+    && groupadd --system --gid $APP_GID $APP_GROUP \
+    && useradd --home-dir /opt/app --system --uid $APP_UID --gid $APP_USER -G $APP_GROUP $APP_USER
```

While technically this will work with Linux shadow tools, we should be specifying the ID instead of name for the `gid` parameter:

```suggestion
    && useradd --home-dir /opt/app --system --uid $APP_UID --gid $APP_UID -G $APP_GROUP $APP_USER
```

---

oh, this was a likely typo. accepting this suggestion.